app/gateway-oss/2.4.x/upgrading.md

```diff
@@ -34,13 +33,10 @@ If the command is successful, and no migration ran
 $ kong reload [-c configuration_file]
```

Comment:
For some reason it won't let me add a comment to line 30, but the link on that line should be updated to `https://docs.konghq.com/gateway-oss/2.3.x/cli/#kong-reload`